osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSpinner.cs

```diff
@@ -178,14 +178,11 @@ protected override void UpdateAfterChildren()
         {
             base.UpdateAfterChildren();
 
-            circle.Rotation = Disc.Rotation;
             Ticks.Rotation = Disc.Rotation;
             spmCounter.SetRotation(Disc.RotationAbsolute);
 
             float relativeCircleScale = Spinner.Scale * circle.DrawHeight / mainContainer.DrawHeight;
             Disc.ScaleTo(relativeCircleScale + (1 - relativeCircleScale) * Progress, 200, Easing.OutQuint);
-
-            symbol.RotateTo(Disc.Rotation / 2, 500, Easing.OutQuint);
```

Comment:
this removes the current extra visual cue that the object is a spinner, making it even more like a normal hitcircle. not sure if this is a direction we want to go.

Reply:
I don't really have an opinion on this. @Hubza

Reply:
Especially with HD where only the center is visible and thus you don't even know if you spin.